Makefile

```diff
@@ -394,3 +394,13 @@ else
 		-r $(REPO_OWNER)/$(REPO_NAME) \
 		-Z > codecov-upload.log
 endif
+
+## -- Target for maintaining consistent crd copies --
+
+.PHONY: consistent-crds
+## Copy crd from manifests/ and cr from examples/nodejs_postgresql to deploy/crds/
+consistent-crds:
+	$(Q)cd ./deploy/crds/ && ln -srf ../../manifests-upstream/${OPERATOR_VERSION}/servicebindingrequests.apps.openshift.io.crd.yaml \
```

Comment:
@Avni-Sharma We should ensure consistent crd at 3 locations:
[1] `deploy/`
[2] `manifests-upstream/`
[3] `manifests/`

There should be changes to ensure crds are consistent in `manifests/` directory as well

Reply:
I don't think it is feasible to trigger changes to manifests/ . It should be manual because it would be taken up by the ART system.  It would automatically trigger a build for the OSBS build. wdyt @sbose78 . Should we be copying it to manifets/ as well?

Reply:
adding another target. (to be used for OS releases)